src/it/multimodule/singleparent/cliwithoutplugin/pom.xml

```diff
@@ -4,10 +4,26 @@
 	<groupId>@project.groupId@</groupId>
 	<artifactId>@project.artifactId@.integration.multimodule.singleparent.cliwithoutplugin</artifactId>
 	<version>@project.version@</version>
-	<packaging>pom</packaging>
+    <packaging>pom</packaging>
+
+    <properties>
+        <maven-compiler-plugin.version>3.10.1</maven-compiler-plugin.version>
+    </properties>
+
+    <build>
```

Comment:
Revert these changes (all poms) by specififying source version in settings.xml

Reply:
How to do mean this?